### PR TITLE
Add `Style<T>` Docs

### PR DESCRIPTION
**Pull request by brminnick**
This PR adds docs for `CommunityToolkit.Maui.Markup.Style<T>`

### CODE REVIEW

**Comment by PRMerger-2** [comment]
Microsoft authors of articles need to make updates through the private repository, so the updates can be staged and validated by current validation rules. After you review staging and fix all validation issues, you can add the sign-off comment to let the PR reviewers know the updates are ready to be merged. 

It's easy to access the private repo version of your articles in GitHub. Just add *-pr* to the URL, as follows: 

 &nbsp;Public repo URL: `https://github.com/MicrosoftDocs/{docs-repo}` 
 &nbsp;Private repo URL: `https://github.com/MicrosoftDocs/{docs-repo-pr}` 

 We have closed this pull request. Please resubmit the updates through the private repo. For more information, see the [contributor guide](https://docs.microsoft.com/contribute/).

**Comment by PRMerger16** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 File Change Percent | This PR contains file(s) with more than 20% file change.

**Comment by PRMerger13** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 File Change Percent | This PR contains file(s) with more than 20% file change.

**Comment by PRMerger12** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 File Change Percent | This PR contains file(s) with more than 20% file change.

File: docs/maui/markup/extensions/style.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+public Style<T> Add(params Behavior[] behaviors);
+public Style<T> Add(params TriggerBase[] triggers);
+```
+
+The `Add` methods can be used like so:
+
+```csharp
+new Label
+{
+    Style = new Stye<Label>()
```
Comment (jfversluis) [inline docs/maui/markup/extensions/style.md:76]:
```suggestion
    Style = new Style<Label>()
```

**Review by jfversluis — APPROVED** [review]

**Comment by PRMerger3** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 File Change Percent | This PR contains file(s) with more than 20% file change.

**Pull request merged**